catalog/task/run-file-updates/0.2/run-file-updates.yaml

```diff
@@ -50,10 +50,12 @@ spec:
           repo=$(echo "${item}" | jq -rc ".repo")
           ref=$(echo "${item}" | jq -rc '.ref // "main"')
           paths=$(echo "${item}" | jq -rc '.paths // []')
+
+          updatedPaths=$(update-paths -p "${paths}" -f $(workspaces.data.path)/snapshot_spec.json)
```

Comment:
The snapshot spec file should definitely be a parameter of the task - especially now that we're using a subdir in all the pipelines to store it.

Reply:
sure. fixed.